metaflow_extensions/utils.py

```diff
@@ -53,38 +56,43 @@ def running_in_local_env() -> bool:
     )
 
 
-def local_pip_install(project_path: Path) -> None:
-    """`pip install` local package at `project_path`."""
-    logging.info(f"pip installing @ {project_path} for {sys.executable}.")
+def is_mflow_conda_environment() -> bool:
+    """True if current process is a Metaflow Conda environment."""
+    return "--environment conda" in " ".join(sys.argv)
```

Comment:
I'd pedantically point out some overlap with `is_task_local`, could refactor slightly:

```suggestion
def flow_has_option(option_name, option_value):
    return f"--{option_name} {option_value}" in " ".join(sys.argv)

def is_mflow_conda_environment() -> bool:
    """True if current process is a Metaflow Conda environment."""
    return flow_has_option("environment", "conda")
```

whilst changing `is_task_local` to:

```python
def is_task_local() -> bool:
    """True if process is running on the same machine as the local orchestrator."""
    _flow_with_option = partial(flow_has_option, "with")
    return not any(map(_flow_with_option, REMOTE_RUNTIMES))
```

Reply:
I'll leave that one for #22